selfdrive/car/honda/values.py

```diff
@@ -613,6 +615,9 @@ class CAR:
       b'38897-TBA-A110\x00\x00',
       b'38897-TBA-A020\x00\x00',
     ],
+    (Ecu.electricBrakeBooster, 0x18da2bf1, None): [
```

Comment:
Adding another ECU will break all other cars being able to fingerprint. Maybe we don't include this one in this PR?

Reply:
I see. But that also means that future releases won't recognize my car which has this ECU (I mean without adding this manually)?

Reply:
I guess we could remove the brakeBooster from the list of essential ECUs: https://github.com/commaai/openpilot/blob/master/selfdrive/car/fw_versions.py#L207

Reply:
Can you add that change to your PR?

Reply:
> I guess we could remove the brakeBooster from the list of essential ECUs: https://github.com/commaai/openpilot/blob/master/selfdrive/car/fw_versions.py#L207

This would be great (as far as I understand). Wouldn't brake backward compatibility but would let EU Civics work.